pkg/reconciler/delivery/table_test.go

```diff
@@ -162,3 +177,19 @@ func withTraffic(field string, nameValuePairs ...pair) RouteOption {
 		panic(fmt.Errorf("withTraffic field can only be 'spec' or 'status'"))
 	}
 }
+
+// assertEventQueued returns a function that is used for PostConditions checking
+// its main purpose is to test whether events are properly enqueued
+func assertEventQueued(key, want string) func(*testing.T, *TableRow) {
+	return func(t *testing.T, r *TableRow) {
```

Comment:
The tableRow can store OtherTestData which could be used to store the event queued or it could be placed in the context.

https://github.com/googleinterns/knative-continuous-delivery/blob/b01e6028adfcdfdf667d9357430f65d8e785c878/vendor/knative.dev/pkg/reconciler/testing/table.go#L100